dist/mk-docker-opts_tests.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -e
+
+# Check help text
```

Comment:
I'm impressed by the effort, but I don't think we need to check the help output. Focusing on the generated output should be fine. Keeping the help output/test in sync doesn't seem worth the effort.